controllers/provider-alicloud/hack/generate-code

```diff
@@ -49,5 +49,5 @@ bash "${PROJECT_ROOT}"/vendor/k8s.io/code-generator/generate-internal-groups.sh
   github.com/gardener/gardener-extensions/controllers/provider-alicloud/pkg/apis \
   github.com/gardener/gardener-extensions/controllers/provider-alicloud/pkg/apis \
   "config:v1alpha1" \
-  --extra-peer-dirs=github.com/gardener/gardener-extensions/controllers/provider-alicloud/pkg/apis/config,github.com/gardener/gardener-extensions/controllers/provider-alicloud/pkg/apis/config/v1alpha1,k8s.io/apimachinery/pkg/apis/meta/v1,k8s.io/apimachinery/pkg/conversion,k8s.io/apimachinery/pkg/runtime \
+  --extra-peer-dirs=github.com/gardener/gardener-extensions/controllers/provider-alicloud/pkg/apis/config,github.com/gardener/gardener-extensions/controllers/provider-alicloud/pkg/apis/config/v1alpha1,k8s.io/apimachinery/pkg/apis/meta/v1,k8s.io/apimachinery/pkg/conversion,k8s.io/apimachinery/pkg/runtime, ithub.com/gardener/gardener-extensions/controllers/provider-alicloud/pkg/apis/config/v1alpha1 \
```

Comment:
```suggestion
  --extra-peer-dirs=github.com/gardener/gardener-extensions/controllers/provider-alicloud/pkg/apis/config,github.com/gardener/gardener-extensions/controllers/provider-alicloud/pkg/apis/config/v1alpha1,k8s.io/apimachinery/pkg/apis/meta/v1,k8s.io/apimachinery/pkg/conversion,k8s.io/apimachinery/pkg/runtime,github.com/gardener/gardener-extensions/controllers/provider-alicloud/pkg/apis/config/v1alpha1 \
```

Also do we need the change in this file as I guess there is no diff in the generated code?